fixtures/stairville/xbrick-quad-16x8w-rgbw.json

```diff
@@ -0,0 +1,797 @@
+          "effectName": "Sound controlled (slow)",
+          "comment": "Sound-controlled operation, slow transition programmes 02 to 23",
+          "soundControlled": true,
+          "switchChannels": {
+            "Intensity red / NoFunction": "NoFunction",
+            "Intensity green / NoFunction": "NoFunction",
+            "Intensity blue / NoFunction": "NoFunction",
+            "Intensity white / NoFunction": "NoFunction",
+            "Dimmer / NoFunction": "Dimmer",
+            "Stroboscope effect / NoFunction": "Stroboscope effect",
+            "NoFunction / Programme Speed / Microphone Sensitivity": "Microphone Sensitivity"
+          }
+        },
+        {
+          "dmxRange": [241, 255],
+          "type": "Effect",
+          "effectName": "Sound controlled (fast)",
+          "comment": "Sound-controlled operation, fast transition programmes 02 to 23",
+          "soundControlled": true,
+          "switchChannels": {
+            "Intensity red / NoFunction": "NoFunction",
+            "Intensity green / NoFunction": "NoFunction",
+            "Intensity blue / NoFunction": "NoFunction",
+            "Intensity white / NoFunction": "NoFunction",
+            "Dimmer / NoFunction": "Dimmer",
+            "Stroboscope effect / NoFunction": "Stroboscope effect",
+            "NoFunction / Programme Speed / Microphone Sensitivity": "Microphone Sensitivity"
+          }
+        }
+      ]
+    },
+    "Programme Speed": {
+      "capability": {
+        "type": "EffectSpeed",
+        "speedStart": "slow",
+        "speedEnd": "fast",
+        "helpWanted": "Slow to Fast or Fast to Slow?"
+      }
+    },
+    "Microphone Sensitivity": {
+      "capability": {
+        "type": "SoundSensitivity",
+        "soundSensitivityStart": "low",
+        "soundSensitivityEnd": "high",
+        "helpWanted": "Low to High or High to Low?"
+      }
+    },
+    "NoFunction": {
+      "capability": {
+        "type": "NoFunction"
+      }
+    }
+  },
+  "modes": [
+    {
+      "name": "4-channel",
+      "shortName": "4ch",
+      "channels": [
+        {
+          "insert": "matrixChannels",
+          "repeatFor": ["All"],
+          "channelOrder": "perPixel",
+          "templateChannels": [
+            "Red $pixelKey",
+            "Green $pixelKey",
+            "Blue $pixelKey",
+            "White $pixelKey"
+          ]
+        }
+      ]
+    },
+    {
+      "name": "6-channel",
+      "shortName": "6ch",
+      "channels": [
+        {
+          "insert": "matrixChannels",
+          "repeatFor": ["All"],
+          "channelOrder": "perPixel",
+          "templateChannels": [
+            "Red $pixelKey",
+            "Green $pixelKey",
+            "Blue $pixelKey",
+            "White $pixelKey"
+          ]
+        },
+        "Dimmer",
+        "Stroboscope effect"
+      ]
+    },
+    {
+      "name": "8-channel",
+      "shortName": "8ch",
+      "channels": [
+        "Intensity red / NoFunction",
```

Comment:
Can the matrix be used here, or does it need to remain as a simple `colorIntensity`?

Reply:
Not sure what you mean.